lisa/base_tools/wget.py

```diff
@@ -81,10 +81,13 @@ def get(
                 f" stdout: {command_result.stdout}"
             )
         actual_file_path = self.node.execute(
-            f"ls {download_file_path}", shell=True, sudo=sudo
+            f"ls {download_file_path}",
+            shell=True,
+            sudo=sudo,
+            expected_exit_code=0,
+            expected_exit_code_failure_message=f"File {actual_file_path} "
+                                                "doesn't exist."
```

Comment:
Can you please setup `black` and `isort`? And then run `make check` to verify our formatting